specific/cmo_sale/views/sale_view.xml

```diff
@@ -11,19 +53,28 @@
                     <field name="project_number" string="Project Number" attrs="{'invisible': [('state','not in',('draft','sent'))]}" readonly="1"/>
                 </field>
                 <xpath expr="///field[@name='project_id']" position="before">
-                    <field name="project_related_id"/>
-                    <field name="event_date_description" style="width:150%"/>
-                    <field name="venue_description" style="width:150%"/>
+                    <field name="project_related_id" attrs="{'readonly': [('order_type','=','sale_order')]}"/>
+                    <field name="event_date_description" style="width:150%" attrs="{'readonly': [('order_type','=','sale_order')]}"/>
+                    <field name="venue_description" style="width:150%" attrs="{'readonly': [('order_type','=','sale_order')]}"/>
                     <field name="amount_before_management_fee" invisible="1"/>
                 </xpath>
                 <xpath expr="///field[@name='project_id']" position="attributes">
                     <attribute name="invisible">1</attribute>
                 </xpath>
                 <field name="client_order_ref" position="after">
-                    <field name="quote_ref_id"/>
+                    <field name="quote_ref_id" options="{'no_open':True,'no_create':1,'no_create_edit':1}"/>
                 </field>
+                <xpath expr="//page[@string='Other Information']/group/group/field[@name='user_id']" position="before">
+                    <field name="approval_id"/>
+                </xpath>
                 <field name="note" position="replace">
```

Comment:
Try not use replace, use invisible. It can break code if some code depend on it.
But if you try to move this block to another location, it is OK.